core/io/udp_server.h

```diff
@@ -51,7 +51,7 @@ class UDPServer : public Reference {
 			return (ip == p_other.ip && port == p_other.port);
 		}
 	};
-	uint8_t recv_buffer[PACKET_BUFFER_SIZE];
+	uint8_t recv_buffer[PACKET_BUFFER_SIZE] = { 0 };
```

Comment:
I would recommend not initializing the buffers when it can be avoided, because this just goes straight to the binary (large amount of zeros), making it bigger.